opencti-platform/opencti-front/src/private/Root.tsx

```diff
@@ -128,11 +132,12 @@ interface RootComponentProps {
 
 const RootComponent: FunctionComponent<RootComponentProps> = ({ queryRef }) => {
   const queryData = usePreloadedQuery(rootPrivateQuery, queryRef);
-  const { me, settings, entitySettings, schemaSCOs, schemaSDOs, schemaSROs } = queryData;
+  const { me, settings, entitySettings, schemaSCOs, schemaSDOs, schemaSROs, schemaRelationsTypesMapping } = queryData;
   const schema = {
     scos: schemaSCOs.edges.map((sco) => sco.node),
     sdos: schemaSDOs.edges.map((sco) => sco.node),
     sros: schemaSROs.edges.map((sco) => sco.node),
+    schemaRelationsTypesMapping,
```

Comment:
Could it be a map instead of a list ? like this :  
`new Map(schemaRelationsTypesMapping.map((n) => [n.key, n.values]));`
So that in `resolveRelationsTypes`, we could direcly get the values by key